pkg/virt-controller/watch/pool.go

```diff
@@ -1215,9 +1236,12 @@ func (c *PoolController) updateStatus(origPool *poolv1.VirtualMachinePool, vms [
 	}
 
 	pool.Status.Replicas = int32(len(vms))
+	if pool.Status.Replicas != pool.Status.ReadyReplicas {
+		pool.Status.ReadyReplicas = int32(len(c.filterReadyVMs(vms)))
```

Comment:
Why not always assign `ReadyReplicas`?

Reply:
```diff
+  pool.Status.ReadyReplicas = 0
```
@feitnomore I was referring to setting the value to 0.  It's fine to set ReadyReplicas to the current state then compare to the desired state `pool.Status.Replicas`.

Reply:
If I always set the pool.Satus.ReadyReplicas, I assume I don't have to compare anything, because right after it, I'll compare pool to the origPool that was on cluster, and in case different, I'll update status. Right?

```
	if !equality.Semantic.DeepEqual(pool.Status, origPool.Status) {
		err := c.statusUpdater.UpdateStatus(pool)
		if err != nil {
			return err
		}
	}
```